### PR TITLE
[Warmfix] Issue:146 remove direct donation

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Problem
Previous binary was rejected by Apple as LF is not a charity (though is a Non-Profit).  They asked us to change the Donation working and functionality

## Issue
#146 

## Approach
The `Support Litecoin Foundation` button fires a sheet with and embedded webpage that the user can use to send grants to Litecoin Foundation

## Steps
- Removed Donation code
- Added more SwiftUI Code
- Added some Test Stubs
- Check the LTC address matches 

## Screenshot/GIF
|Resulting Action in Send|
|------------------------|
|SupportLF|

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,7 +26,6 @@ playground.xcworkspace
 #
 # Add this line if you want to avoid checking in source code from Swift Package Manager dependencies.
 # Packages/
-.build/
```
Comment (mosadialiou) [inline .gitignore:29]: put it back

**Review by mosadialiou — CHANGES_REQUESTED** [review]
We mustn't have changes related to ReduxState in this PR. The rebase is not good.

**Comment by kcw-grunt** [comment]
There are issues with the rebase...closing

**Comment by mosadialiou** [comment]
> There are issues with the rebase...closing

OK but take my review into account

**Pull request closed without merging**